### ISSUE TITLE
Card explorer: click split card image to view popup at correct orientation.

### ISSUE THREAD

**Issue by lodici** [magarena/magarena#782]

**Comment by lodici** [comment]
Need a way to identify this style as a 90 degree rotate does not work so well -
![screenshot00630](https://user-images.githubusercontent.com/2443519/27480822-a8d3412c-5811-11e7-919e-3c2c504d8fd1.jpg)

**Comment by lodici** [comment]
The new style was introduced in AKH (http://www.manaleak.com/mtguk/2017/04/a-brief-history-of-split-cards-and-the-amonkhet-aftermath-by-dante-gutierrez/) so for now can just check whether a split-card belongs to AKH to determine whether to rotate -90 degrees.

**Comment by ShawnieBoy** [comment]
This isn't simply a new style - it's specific to cards with the Aftermath ability. I'd suggest refering it to as `isAftermathSplitCard` as it's a mechanical choice not a style one. Aftermath will also be appearing in Hour of Devastation - this will need to be updated when that set is released next month.

**Comment by ShawnieBoy** [comment]
It could be possible (Although I'd think unlikely) that Hour of Devastation may have split cards with and without the Aftermath ability. Will have to wait and see.

**Comment by lodici** [comment]
Yes, ``isAftermathSplitCard`` is clearer although if it represents an ability it would be better to query the card object with something like ``card.hasAbility(Ability.Aftermath) && card.isSplitCard()`` so as to remove the brittle dependency on checking a set. However, I see nothing regarding the aftermath ability currently in ``MagicCardDefinition``.

**Comment by ShawnieBoy** [comment]
The problem with .`hasAbility` is that it will only catch information if abilities are loaded beforehand, and if a card is actually implemented. As split cards aren't currently implemented, aftermath isn't either.

**Comment by ShawnieBoy** [comment]
A similar situation is with Devoid and 'Level up' cards while displaying cardbuilder frames - cards in card explorer have no abilities present, so have no ability dependant frames.

**Comment by lodici** [comment]
> The problem with .hasAbility is that it will only catch information if abilities are loaded beforehand, and if a card is actually implemented. As split cards aren't currently implemented, aftermath isn't either.

Remind me again why loading abilities is such a pain. Is it because it needs to read groovy which is slow? Thinking out loud, could the code for extracting abilities be moved to ScriptBuilder and the ability keywords added to the script file so that they can be read into the MagicCardDefinition object on creation as a simple set of flags.

**Comment by ShawnieBoy** [comment]
I think it was removed due to the amount of time it takes when opening the app. I use two different .bat files to open with and without depending on what I'm working on.
Ability keywords are in the script file, but unimplemented abilities/cards will not be caught by` loadAbilities()` anyway.

**Comment by melvinzhang** [comment]
For current aftermath cards, Aftermath is part of the effect properly. We can check it there with an isAftermathSplitCard method in MagicCardDefinition, as `effectProperty.contains("Aftermath")`. Note that only the bottom half of the card has Aftermath.

**Comment by lodici** [comment]
Ideally, the UI should not be making the decision as it does now via the set-check hack. A simple call to an isAftermatchSplitCard() method of MagicCardDefintion would be much more preferable.  @melvinzhang I think you should implement your solution.

**Comment by ShawnieBoy** [comment]
As Aftermath is actually an ability (Like Fuse, in the way that it only appears on Split cards) - It would make sense to update the scriptbuilder and reparse the split cards in Amonkhet as well
